src/transformers/utils/doc.py

```diff
@@ -1140,6 +1142,15 @@ def docstring_decorator(fn):
         )
         if real_checkpoint is not None:
             code_sample = FAKE_MODEL_DISCLAIMER + code_sample
+        if revision is not None:
+            if "refs/pr/" not in revision:
+                raise ValueError(
+                    f"The provided revision '{revision}' is incorrect. It should point to"
+                    " a pull request reference on the hub like 'refs/pr/6'"
+                )
+            code_sample.replace(
+                f'from_pretrained("{checkpoint}")', 'from_pretrained("{checkpoint}", revision="{revision}")'
```

Comment:
```suggestion
                f'from_pretrained("{checkpoint}")', f'from_pretrained("{checkpoint}", revision=f"{revision}")'
```
think you are missing some f strings here